test/python/test_tf2ngraph_script.py

```diff
@@ -163,6 +163,8 @@ def test_command_line_api(self, inp_format, inp_loc, out_node_name,
             # Comparing with expected value
             assert np.isclose(res1, exp).all()
 
+    #TODO: Unskip this test
+    @pytest.mark.skip(reason="Does not work with tf v1.15.0")
```

Comment:
@sayantan-nervana fyi. This test is being skipped.

Reply:
@ashahba : If its working now lets reenable it. If not, can you please create a ticket to reenable it, so that it is tracked.

Reply:
Let me try uncommenting this and running it and if it fails, I'll file a ticket or look into a fix.

Reply:
It's fixed now